packages/create-plugin/src/constants.ts

```diff
@@ -43,7 +43,7 @@ export enum PLUGIN_TYPES {
 // and will be available to use in the templates.
 // Example: "@grafana/ui": "{{ grafanaVersion }}"
 export const EXTRA_TEMPLATE_VARIABLES = {
-  grafanaVersion: '10.0.3',
+  grafanaVersion: '10.2.3',
```

Comment:
This isn't a straight-forward change I'm afraid. You'll notice the CI is now failing due to peer dependency issues. As such can you please revert this?